### PR TITLE
Add themes, light theme

### PR DESCRIPTION
**Pull request by sigvef**
image

### CODE REVIEW

File: nin/frontend/app/scripts/controllers/main.js [hunk 1]
```diff
@@ -4,6 +4,27 @@
...
+      $scope.themes = [
+        'dark',
+        'light'
+      ];
+
+      $scope.selectedTheme = localStorage.getItem('selectedTheme') || 'dark';
+      $scope.selectTheme = function(theme) {
+        var foundTheme = false;
+        for(var i = 0; i < $scope.themes.length; i++) {
+          if($scope.themes[i] == theme) {
```
Comment (aleksanb) [inline nin/frontend/app/scripts/controllers/main.js]: Can be replaced with .indexOf != -1 ?

File: nin/frontend/app/scripts/controllers/main.js [hunk 2]
```diff
@@ -113,6 +134,17 @@
           ]
         },
         {
+          name: 'Theme',
+          items: [
+            {name: 'Dark', click: function() {
+              $scope.selectTheme('dark');
```
Comment (aleksanb) [inline nin/frontend/app/scripts/controllers/main.js]: $scope.selectTheme vs commands.selectTheme for decoupling?

**Comment by aleksanb** [comment]
Awesome, this white theme is really nice.
Not quite related to this, but i just noticed how horrible our layers look when i switched to the white design:
screenshot from 2015-08-08 00 06 53

File: nin/frontend/app/scripts/controllers/main.js (hunk 1 above)
Comment (sigvef) [inline nin/frontend/app/scripts/controllers/main.js]: That works with arrays too? cool

**Comment by sigvef** [comment]
> Perhaps we could force the layers to be one of a set of nice predetermined colors.

Yes, I've been thinking the same thing. Coincidentally, this is what FL does :P

**Comment by sigvef** [comment]
ping, added predetermined colors.

**Comment by cristeahub** [comment]
Can confirm this looks more pleasing on the eyes for some :+1:

**Pull request merged**